pkgs/by-name/go/golem/package.nix

```diff
@@ -0,0 +1,55 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  openssl,
+  protobuf,
+  redis,
+  fontconfig,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "golem";
+  version = "1.0.23";
+
+  src = fetchFromGitHub {
+    owner = "golemcloud";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-ZvqlWavntlzQLUuTZG+pCeEZTT+DTP+QjiRgXu/9TvQ=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    protobuf
+  ];
+  buildInputs = [
+    fontconfig
+    openssl.dev
+  ];
+
+  # Tests are failing in the sandbox because of some redis integration tests
+  doCheck = false;
+  checkInputs = [ redis ];
+
+  # Required for golem-wasm-rpc's build.rs to find the required protobuf files
+  # https://github.com/golemcloud/wasm-rpc/blob/v1.0.6/wasm-rpc/build.rs#L7
+  GOLEM_WASM_AST_ROOT = "/build/cargo-vendor-dir/golem-wasm-ast-1.0.1";
+  # Required for golem-examples's build.rs to find the required Wasm Interface Type (WIT) files
+  # https://github.com/golemcloud/golem-examples/blob/v1.0.6/build.rs#L9
+  GOLEM_WIT_ROOT = "/build/cargo-vendor-dir/golem-wit-1.0.1";
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "cranelift-bforest-0.108.1" = "sha256-WVRj6J7yXLFOsud9qKugmYja0Pe7AqZ0O2jgkOtHRg8=";
+    };
+  };
+
```

Comment:
```suggestion
  };
  
  nativeInstallCheckInputs = [
    versionCheckHook
  ];
  versionCheckProgramArg = [ "--version" ];
  doInstallCheck = true;

```

Reply:
Have you tried this ?

Reply:
The build just failed locally, changed it to `"out"`

Reply:
Oh I didn't see that it was above.
Can you put the whole "check" block (from `# Tests are failing in the sandbox because of some redis integration tests` to `doInstallCheck = true;` just above `passthru` please ?

Reply:
moved it